### PR TITLE
Update action runtime to node 16

### PR DESCRIPTION
**Pull request by sibiraj-s**
- update action runtime to node 16
- use getBooleanInput and getMultilineInput from actions/code

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
 import {inspect} from 'util'
-import * as utils from './utils'
 
 async function run(): Promise<void> {
   try {
     const inputs: Inputs = {
       token: core.getInput('token'),
       path: core.getInput('path'),
-      addPaths: utils.getInputAsArray('add-paths'),
+      addPaths: core.getMultilineInput('add-paths'),
```
Comment (peter-evans) [inline src/main.ts:10]: `getMultilineInput` is not a replacement for my implementation. It does not parse comma-separated values. This action used comma-separated values in a much older version and I've kept that functionality for backwards compatibility. I cannot remove this without causing a breaking change. At some point in the future I may consider it when bumping the major version, but for now please revert this back.

File: src/main.ts [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
       commitMessage: core.getInput('commit-message'),
       committer: core.getInput('committer'),
       author: core.getInput('author'),
-      signoff: core.getInput('signoff') === 'true',
+      signoff: core.getBooleanInput('signoff'),
```
Comment (peter-evans) [inline src/main.ts:14]: I'm happy to switch to `getBooleanInput` because there is no risk of a breaking change for users here. 👍

**Review by peter-evans — CHANGES_REQUESTED** [review]
Hi @sibiraj-s 

Thank you for contributing.
Please see my comment about reverting `getMultilineInput`, but the other changes look good to me.

File: src/main.ts (hunk 1 above)
Comment (sibiraj-s) [reply src/main.ts:10]: Ah, I didn't notice that. Will revert this.

Comment (sibiraj-s) [reply src/main.ts]: Reverted now.

**Review by peter-evans — APPROVED** [review]

**Comment by peter-evans** [comment]
Thank you!

**Comment by peter-evans** [comment]
Unfortunately, I had to revert this commit because it caused [an issue](https://github.com/peter-evans/create-pull-request/issues/1077) for users on GitHub Enterprise.

**Comment by sibiraj-s** [comment]
Apologies. I had no idea that Github Enterprise didn't support it. It was quite a while since node16 was released, and I failed to account enterprise runners.

**Comment by peter-evans** [comment]
No worries. It was my bad for not checking. Appreciate the effort to contribute. 👍

**Comment by sibiraj-s** [comment]
Thanks.

And, node 12 is coming to its EOL next month. And many [official](https://github.com/actions/github-script/releases) [actions](https://github.com/actions/setup-node/releases) started releasing new majors for node 16. Any thoughts on v4?

**Review by Tuneabq — COMMENTED** [review]
,😃✅💯

**Pull request merged**